src/argilla/server/contexts/datasets.py

```diff
@@ -56,6 +56,9 @@
 
 LIST_RECORDS_LIMIT = 20
 
+VISIBLE_FOR_ANNOTATORS_ALLOWED_ROLES = [UserRole.admin, UserRole.annotator]
+NOT_VISIBLE_FOR_ANNOTATORS_ALLOWED_ROLES = [UserRole.admin]
```

Comment:
Just a nit, but maybe we could rename those to something like `METADATA_VISIBLE_ALLOWED_ROLES` and `METADATA_NOT_VISIBLE_ALLOWED_ROLES`, just mentioning this because I recall we decided not to name it `visible_for_annotators` (even this may be the case in the Python client exposed to the user), so maybe the rename makes it clearer in the backend, WDYT?

Reply:
We are actually exposing in the API a "virtual/calculated" boolean attribute that is `visible_for_annotators`. I can add `METADATA_PROPERTY` but then is really long like `METADATA_ PROPERTY_VISIBLE_FOR_ANNOTATORS_ALLOWED_ROLES`.